README.rst

```diff
@@ -27,53 +27,97 @@ Documentation
 Development Workflow
 ********************
 
-One Time Setup
-==============
+Initial Setup: steps and confirmation
+======================================
 .. code-block::
 
   # Clone the repository
   git clone git@github.com:edx/commerce-coordinator.git
   cd commerce-coordinator
 
-  # Set up a virtualenv using virtualenvwrapper with the same name as the repo and activate it
-  mkvirtualenv -p python3.8 commerce-coordinator
+  # install mysql in your local directory
+  brew install mysql
+
+  # Set up a virtualenv and activate it
+  python3 -m venv .venv
+  source .venv/bin/activate
+
+  # Install/update the dev requirements (run inside the venv)
+  make requirements
+  # If you get 'Unsupported architecture' errors above, instead use: CFLAGS='-D__x86_64__' make requirements
+
+  # run commerce-coordinator locally (run inside the venv)
+  python manage.py runserver localhost:8000 --settings=commerce_coordinator.settings.local
+
+  # You should see output ending with "Quit the server with CONTROL-C"
+  # In your browser, hit the URL: http://localhost:8000/demo_lms/test/
+  # You should see JSON output indicating that two receivers were called, one successful, and one with exception/traceback information.
+  # In the shell where the server is running you should see log output indicating that two test receivers were called with the sender argument "Something".
 
 
 Every time you develop something in this repo
 =============================================
 .. code-block::
 
-  # Activate the virtualenv
-  workon commerce-coordinator
-
   # Grab the latest code
   git checkout main
   git pull
 
-  # Install/update the dev requirements
-  make requirements
+  # Activate the virtualenv
+  source .venv/bin/activate
 
-  # Run the tests and quality checks (to verify the status before you make any changes)
-  make validate
+  # Install/update the dev requirements (run inside the venv)
+  make requirements
+  # If you get 'Unsupported architecture' errors above, instead use: CFLAGS='-D__x86_64__' make requirements
 
   # Make a new branch for your changes
   git checkout -b <your_github_username>/<short_description>
 
-  # Using your favorite editor, edit the code to make your change.
-  vim …
-
-  # Run your new tests
-  pytest ./path/to/new/tests
-
-  # Run all the tests and quality checks
+  # Run the tests and quality checks (before and after your changes)
```

Comment:
(I removed some bits that are typical development process, and not typically listed out in a README)